src/internal/provider/protocols.go

```diff
@@ -14,14 +12,12 @@ type ManifestMetadata struct {
 	ProtocolVersions []string `json:"protocol_versions"`
 }
 
-func GetProtocols(ctx context.Context, manifestDownloadUrl string) ([]string, error) {
-	if manifestDownloadUrl == "" {
-		slog.Warn("Could not find manifest file, using default protocols")
-		return []string{"5.0"}, nil
+func (p Provider) GetProtocols(manifestDownloadUrl string) ([]string, error) {
+	contents, err := p.Github.DownloadAssetContents(manifestDownloadUrl)
+	if err != nil {
+		return nil, err
 	}
 
```

Comment:
It's often the case that the manifest doesnt exist, and in these cases we should assume that it is 5.0, 

can you add this logic back in please?

Reply:
I moved this logic to the caller, it's not removed.  IMO it's cleaner to have it in there, that's where we validate / parse the release information.

Reply:
I see this is moved there so I'm happy with that 👍